doc/spec/transformations.md

```diff
@@ -223,4 +223,22 @@ Examples:
 {"op": "core/rename-column",
  "args": {"columnName": "c4",
           "newColumnTitle": "New Title"}}
-```
\ No newline at end of file
+```
+
+### core/generate-geopoints
+
+* args
+  * columnNameLat (string): The column containing latitude data
+  * columnNameLong (string): The column containing longitude data
+  * columnTitleGeo (string): The title of the geopoint column to be generated
+  * columnTypeLat (string): The client data type of the latitude column
+  * columnTypeLong (string): The client data type of the longitude column
```

Comment:
Why are these needed? When the backend knows the `columnNameLat/Long` it can figure out the types as well.